### PR TITLE
Use Prettier JS

### PR DESCRIPTION
**Pull request by flovilmart**
Because it's awesome!

It's also combined with pre-commit hooks so your code will always be pretty!

### CODE REVIEW

**Comment by flovilmart** [comment]
@acinader no more 'coding style' arguments!

File: package.json [hunk 1]
```diff
@@ -96,5 +98,18 @@
...
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "{src,spec}/**/*.js": [
+      "prettier --write",
+      "flow",
+      "eslint --cache ./",
+      "git add"
```
Comment (acinader) [inline package.json]: @flovilmart a bad side effect of this change is that if you commit with unstaged changes, they'll actually get committed.

Comment (flovilmart) [reply package.json]: No they don’t only the list of files that are staged are passed as an argument.

Husky passes the list of files to the commandsz

Comment (acinader) [reply package.json]: In my case, I staged part of a file, but it would end up committing all the changes in the file, not just the parts I staged.

Comment (flovilmart) [reply package.json]: You’re right, still, this feels like a corner case

Comment (acinader) [reply package.json]: I can live with it

**Pull request merged**